conscious/conscious/CategoryTableViewCell.swift

```diff
@@ -16,7 +16,7 @@ class CategoryTableViewCell: UITableViewCell {
     override func awakeFromNib() {
         super.awakeFromNib()
 
-        tapGestureRecognizer = UITapGestureRecognizer(target: self, action: "onTap")
+        tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(CategoryTableViewCell.onTap))
```

Comment:
I see you have updated your xcode to the latest.  This doesn't build for me, so in order to use this, ill need to update my phone, which will cause an issue for me at work, because they don't want to update their build servers yet.  Any chance we can hold off updating to the swift 2.2 selector syntax?